.gitignore

```diff
@@ -74,3 +74,6 @@ embedded_zookeeper/
 **/operation_logs/
 **/server_operation_logs/
 **/engine_operation_logs/
+conf/log4j2.properties
+conf/kyuubi-defaults.conf
+conf/kyuubi-env.sh
```

Comment:
I think it's a good idea to exclude those files then we can debug with specific configurations in local easily